### DISCUSSION TITLE
Bug

### DISCUSSION THREAD

**Discussion by shahzada83** [livewire/livewire#5500] [1. Help]
With the release of v2.10.8, the existing application on 2.10.7 is breaking.

We are getting Livewire/message/{component . full . path} 404 Not found error in the console.

And 404 Error popup model appears on the application.

Reverted back to 2.10.7 in composer.json file, all is working as previously.

**Comment by joshhanley** [comment]
@shahzada83 please search the discussions first before posting, as this problem has been posted numerous times 🙂 see #5464 for more details. And see this open PR #5490 which should fix the issue.
